content/homepage/index.yml

```diff
@@ -2,6 +2,7 @@ title: Homepage
 main:
   - /_shared/header.md
   - body.md
+  - imageblock.md
```

Comment:
Can you put in only on our testing pages so we don't break what the content folks already done?